common/resources/assets/negativity/Negativity_fr_FR.properties

```diff
@@ -2,16 +2,16 @@ prefix=&a[&2Negativity&a]
 
 report_wait=Merci d'attendre une seconde entre 2 reports
 
-invalid_player=%arg% n'est pas un joueur !
-not_forget_player=N'oubliez pas le joueur !
+invalid_player=%arg% n'est pas un joueurs!
```

Comment:
French actually has spaces before exclamation marks (and question marks) so please revert these changes in both fr_FR files.
The added s is also incorrect btw. It was fine before.

Reply:
Oh ok